MIGRATION.md

```diff
@@ -1,5 +1,13 @@
 ## Migration guide
 
+### Migrating from v4 to v5
+
+V6 has dropped support for use through Bower.
+
+If you were already using npm, no changes should be required.
+
+If you have been using Bower or the Origami Build Service, follow [the migration guide on the Origami website](https://origami.ft.com/docs/tutorials/bower-to-npm/).
+
```

Comment:
It might be worth pointing all users to the migration guide, as they may need to upgrade to npm 7 for example https://github.com/Financial-Times/origami-website/pull/363/files#diff-9e78abc904bb5a671d5ea77bf1f59c8d7ad872a0d3990d2f9ab418ad0b077f77R82

```suggestion
Follow [the migration guide on the Origami website](https://origami.ft.com/docs/tutorials/bower-to-npm/).
```